docs/res/intro.md

```diff
@@ -0,0 +1,138 @@
+By calling individual methods of `ApifyClient`, specific clients which target individual API
+resources are created. There are two types of those clients. A client for management of a single
+resource and a client for a collection of resources.
+
+```python
+from apify_client import ApifyClient
+apify_client = ApifyClient('MY-APIFY-TOKEN')
+
+# Collection clients do not require a parameter
+actor_collection_client = apify_client.actors()
+# Create an actor with the name: my-actor
+my_actor = actor_collection_client.create({ name: 'my-actor' })
+# List all of your actors
+actor_list = actor_collection_client.list()['items']
+```
+
+```python
+# Collection clients do not require a parameter
+dataset_collection_client = apify_client.datasets()
+# Get (or create, if it doesn't exist) a dataset with the name of my-dataset
+my_dataset = dataset_collection_client.get_or_create('my-dataset')
+```
+
+```python
+# Resource clients accept an ID of the resource
+actor_client = apify_client.actor('john-doe/my-actor')
+# Fetch the john-doe/my-actor object from the API
+my_actor = actor_client.get()
+# Start the run of john-doe/my-actor and return the Run object
+my_actor_run = actor_client.start();
+```
+
+```python
+# Resource clients accept an ID of the resource
+dataset_client = apify_client.dataset('john-doe/my-dataset')
+# Append items to the end of john-doe/my-dataset
+dataset_client.push_items([{ "foo": 1 }, { "bar": 2 }])
+```
+
+> The ID of the resource can be either the `id` of the said resource,
+> or a combination of your `username/resource-name`.
+
+This is really all you need to remember, because all resource clients
+follow the pattern you see above.
+
+### Nested clients
+
+Sometimes clients return other clients. That's to simplify working with
+nested collections, such as runs of a given actor.
+
+```python
+actor_client = apify_client.actor('john-doe/my-actor')
+runs_client = actor_client.runs()
+# List the last 10 runs of the john-doe/hello-world actor
+actor_runs = runs_client.list(limit=10, desc=True)['items']
+
+# Select the last run of the john-doe/hello-world actor that finished with a SUCCEEDED status
+last_succeeded_run_client = actor_client.last_run(status='SUCCEEDED')
+# Fetch items from the run's dataset
+dataset_items = last_succeeded_run_client.dataset().list_items()['items']
+```
+
+> The quick access to `dataset` and other storages directly from the run
+> client can now only be used with the `last_run()` method, but the feature
+> will be available to all runs in the future.
+
+### Pagination
+
+Most methods named `list` or `list_something` return a pagination dictionary,
+containing keys `items`, `total`, `offset`, `count` and `limit`.
+There are some exceptions though, like `list_keys` or `list_head` which paginate differently.
+The results you're looking for are always stored under `items` and you can use the `limit`
+property to get only a subset of results. Other props are also available, depending on the method.
```

Comment:
A better way to express it would be "Other props can be available depending on the method."